examples/connectivity/plot_adhd_spheres.py

```diff
@@ -0,0 +1,74 @@
+"""
+Extracting brain signal from spheres
+====================================
+
+This example estimates a connectivity between Default Mode Network components
```

Comment:
Before telling us this, tell us that it extract signals on sphere specified by there position in MNI space. That's the goal of the example in terms of learning experience.